### DISCUSSION TITLE
RSC not found on home page with i18n

### DISCUSSION THREAD

**Discussion by wannesmatthys** [vercel/next.js#57739] [App Router]
Hi, I'm creating a multi langual system with my next js application on the App Router.

Everything is working fine on local, but on production when I navigate to the homepage, it always forces a hard refresh.
This gets caused (I think) because the ?_rsc request on the home page always results in a 404 not found.
![image](https://github.com/vercel/next.js/assets/40867324/0815d315-a9b4-4772-9ec2-e1f3de14670c)

Other pages are working fine, and the ?rsc returns successfully. This issue is only on the home page.

This seems to happen when adding i18n config.

Is this a known issue or are there any fixed?

Next: 13.5.6

Kr,
Wannes

**Comment by hyjoong** [comment]
Refer to this link: https://github.com/vercel/next.js/discussions/50518
